django_cookies_samesite/middleware.py

```diff
@@ -19,7 +21,7 @@
 
 
 Cookie.Morsel._reserved['samesite'] = 'SameSite'
-
+CHROME_VALIDATE_REGEX = "Chrome\/((5[1-9])|6[0-6])"
```

Comment:
I would copy the regex for the Chrome browser from the article that you've mentioned in the first comment. This line has some flaws e.g. values like 'Chrome600' will match too, because of the missing space at the end.